Makefile

```diff
@@ -1,10 +1,30 @@
-all:
-	go get github.com/tools/godep
+CWD:=$(shell pwd)
+RUNC_TEST_IMAGE=runc_test
+PROJECT=github.com/opencontainers/runc
+TEST_DOCKERFILE=test_Dockerfile
+
+all: godep
 	godep go build -o runc .
 
+godep:
+	go get github.com/tools/godep
+
+lint: godep
+	go get golang.org/x/tools/cmd/vet
+	godep go vet ./...
+	godep go fmt ./...
+
+runctestimage:
+	docker build -t $(RUNC_TEST_IMAGE) -f $(TEST_DOCKERFILE) .
+
+test: runctestimage
+	docker run --privileged --rm -v $(CWD):/go/src/$(PROJECT) $(RUNC_TEST_IMAGE) make localtest
+
+localtest: godep
+	godep go test -v ./...
+
 install: all
```

Comment:
I think we should take out this dependency on all per @crosbymichael comment in the other PR as install requires sudo and could mess things up.